soh/soh/Enhancements/randomizer/3drando/location_access/locacc_ganons_castle.cpp

```diff
@@ -376,4 +369,54 @@ void RegionTable_Init_GanonsCastle() {
                   EventAccess(&logic->LightTrialClear, {[]{return logic->CanUse(RG_LIGHT_ARROWS);}}),
   }, {}, {});
   }
+
+  /*--------------------------
+  |     TOWER AND ESCAPE     |
+  ---------------------------*/
+  areaTable[RR_GANONS_TOWER_FLOOR_1] = Region("Ganon's Tower Floor 1", "Ganons Castle", {RA_GANONS_CASTLE}, NO_DAY_NIGHT_CYCLE, {}, {}, {
+                  //Exits
+                  Entrance(RR_GANONS_CASTLE_LOBBY,  {[]{return true;}}),
+                  Entrance(RR_GANONS_TOWER_FLOOR_2, {[]{return logic->CanKillEnemy(RE_DINOLFOS, ED_CLOSE, true, 2);}}),
```

Comment:
Do these fights stay cleared when beaten if you leave and return? IIRC they do and if so you need to use Here() on these exits (but not the chest) to cover for 1 age clearing them to make a path for the other age to beat a different floor.